radiography/src/main/java/radiography/ScannableView.kt

```diff
@@ -18,16 +19,24 @@ import radiography.internal.mightBeComposeView
  * Can either be an actual Android [View] ([AndroidView]) or a grouping of Composables that roughly
  * represents the concept of a logical "view" ([ComposeView]).
  */
-public sealed class ScannableView {
+public interface ScannableView {
 
   /** The string that be used to identify the type of the view in the rendered output. */
-  public abstract val displayName: String
+  public val displayName: String
 
   /** The children of this view. */
-  public abstract val children: Sequence<ScannableView>
+  public val children: Sequence<ScannableView>
 
-  public class AndroidView(public val view: View) : ScannableView() {
-    override val displayName: String get() = view::class.java.simpleName
+  public class AndroidView(public val view: View) : ScannableView {
+    override val displayName: String get() {
+      val classSimpleName = view::class.java.simpleName
+      val windowTitle = (view.layoutParams as? WindowManager.LayoutParams)?.title?.toString()
+      if (windowTitle != null) {
+        return "$classSimpleName in $windowTitle window-focus:${view.hasWindowFocus()}"
+      } else {
+        return classSimpleName
```

Comment:
Why not report focus in this case?

Reply:
If there's no window title, then this isn't the root view for an attached window. In which case window focus doesn't make sense. Good question though, should add as comment.